SabberStoneCore/src/CardSets/Standard/DalaranCardsGen.cs

```diff
@@ -1378,10 +1378,8 @@ private static void Rogue(IDictionary<string, Power> cards)
 			// - REQ_MINION_TARGET = 0
 			// --------------------------------------------------------
 			cards.Add("DAL_716", new Power {
-				// TODO [DAL_716] Vendetta && Test: Vendetta_DAL_716
-				//PowerTask = null,
-				//Trigger = null,
-				//PowerTask = new DamageTask(4, EntityType.TARGET, true),
+				PowerTask = new DamageTask(4, EntityType.TARGET),
+				Aura = new AdaptiveCostEffect(0, TriggerType.ZONE, triggerCondition: SelfCondition.HoldingAnotherClassCard)
```

Comment:
@rnilva I'm not sure ZONE is correct, maybe ZONE and DISCARD is needed?  I figured this be harder than it looks.  Needs to be triggered whenever the hand changes.